readme/dev/spec/performance.md

```diff
@@ -30,7 +30,7 @@ On desktop and web (in Chrome), `PerformanceLogger` tasks and marks are added to
 
```

Comment:
Please also add a section about naming convention we should be `filename/description`